test/build/goldens/build-config-dev-env.golden.json

```diff
@@ -85,6 +85,7 @@
             {
               "loader": "/node_modules/extract-text-webpack-plugin/dist/loader.js",
               "options": {
+                "id": 3,
```

Comment:
FYI this got added because we're no longer creating multiple instances of `extract-text-webpack-plugin`.
Instead, we're reusing one plugin instance throughout the entire config file.

According to [their documentation](https://github.com/webpack-contrib/extract-text-webpack-plugin#options), the `"id"` attribute is automatically generated by the plugin.